docker.d/init.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+set -e
+
+#
+# Check for certain variables which should be set
+#
+test $PROJECT_NAME
+test $ENV
+test $COT_PRODUCT
+test $TASKCLUSTER_CLIENT_ID
+test $TASKCLUSTER_ACCESS_TOKEN
+if [ "$ENV" == "prod" ]; then
+  test $ED25519_PRIVKEY
+fi
```

Comment:
In signingscript oremj+catlee landed a fix to [echo missing variables](https://github.com/mozilla-releng/signingscript/commit/5b5691905e9445b4cfa0e033dc3955018f9a6d04#diff-b8e93dc171018744d01390a4b1191c85) to make it easier to debug. Not sure if we want to carry that over here.

Reply:
this is exactly the reason why i would like to move all of the signing script as soon as possible. I'll update the init.sh with that version.